lib/alex_koin/slack.ex

```diff
@@ -3,7 +3,7 @@ defmodule AlexKoin.SlackRtm do
     :alex_koin, :slack_module, Slack.Sends
   )
 
-  @koin_bot_id Application.get_env(:alex_koin, :koin_bot_id)
+  @koin_bot_id Application.get_env(:alex_koin, :koin_bot_id) || ""
```

Comment:
`get_env` supports a [default](https://hexdocs.pm/elixir/Application.html#get_env/3) parameter, which we should probably use instead of the `||`